pageserver/src/tenant/timeline.rs

```diff
@@ -115,6 +114,15 @@ pub struct Timeline {
 
     pub(super) layers: RwLock<LayerMap<dyn PersistentLayer>>,
 
+    //
+    // Set of key ranges which should be covered by image layers to
+    // allow GC to remove old layers. BTreeMap key is start of range (inclusive),
+    // value - end of range (exclusive). This map is constructed by GC
+    // (when it can not remove layer because it is not covered by image layers)
+    // and used by compaction when it checks if new image layer should be created.
+    //
```

Comment:
```suggestion
    /// Set of key ranges which should be covered by image layers to
    /// allow GC to remove old layers.
    ///
    /// BTreeMap key is start of range (inclusive),
    /// value - end of range (exclusive). This map is constructed by GC
    /// (when it can not remove layer because it is not covered by image layers)
    /// and used by compaction when it checks if new image layer should be created.
```

rustdocify, remove excess comment lines.